python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -257,60 +256,87 @@ def repackage_eimage(self, phosim_eimg_file, out_dir='.'):
         sensor.header['RUNNUM'] = sensor.header['OBSID']
-        sensor.header['DATE-OBS'] = \
-            astropy.time.Time(sensor.header['MJD-OBS'], format='mjd').isot
+        sensor.header['MJD-OBS'] = sensor.header['MJD-OBS']
+
+        DATEOBS = astropy.time.Time(sensor.header['MJD-OBS'], format='mjd').isot
+        sensor.header['DATE-OBS'] = DATEOBS
+
+        YEAR, MONTH, DAYTIME = DATEOBS.split('-')
+        DAY = DAYTIME[:2]
+        DAYOBS = '%s%s%s'%(YEAR, MONTH, DAY)
+        sensor.header['DAYOBS'] = DAYOBS
 
-        chip_id = sensor.header['CHIPID']
-        sensor.header['LSST_NUM'] = chip_id
+        # file creation date as mjd
+        sensor.header['MJD'] = astropy.time.Time(sensor.header['DATE'],
+                                                 format='isot').mjd
 
-        parts = chip_id.split('_')
-        raft = parts[0]
-        ccd = parts[1]
-        sensor.header['RAFTNAME'] = raft
-        sensor.header['SENSNAME'] = ccd
+        serial = detectors.getSerial()  # eg. ITL-4400B-029
+        CCD_MANU, CCD_TYPE, CCD_NUM = serial.split('-')
+        sensor.header['LSST_NUM'] = serial
+        sensor.header['CCD_MANU'] = CCD_MANU  # eg. ITL
+        sensor.header['CCD_TYPE'] = CCD_TYPE  # eg. 4400B
+
+        sensor.header['DETSIZE'] = noao_section_keyword(detectors.getBBox())
+        sensor.header['INSTRUME'] = 'lsstCam'
+        sensor.header['TELESCOP'] = 'LSST'
+
+        sensor.header['TELCODE'] = TELCODE
+        sensor.header['CONTRLLR'] = CONTRLLR
+        SEQNUM = int(sensor.header['RUNNUM'][-6:])
+        sensor.header['SEQNUM'] = SEQNUM
+
+        OBSID = "%s_%s_%s_%s"%(TELCODE, CONTRLLR, DAYOBS, str(SEQNUM).zfill(6))
+        sensor.header['OBSID'] = OBSID
 
         sensor.header['TESTTYPE'] = 'PHOSIM'
         sensor.header['IMGTYPE'] = 'SKYEXP'
-        sensor.header['MONOWL'] = -1
-
-        # Add boresight pointing angles and rotskypos (angle of sky
-        # relative to Camera coordinates) from which obs_lsst can
-        # infer the CCD-wide WCS.
-        sensor.header['RATEL'] = sensor.header['RA_DEG']
-        sensor.header['DECTEL'] = sensor.header['DEC_DEG']
-
-        # The rotation angle has different names in different versions
-        try:
-            sensor.header['ROTANGLE'] = sensor.header['ROTANGZ']
-        except KeyError:
-            try:
-                sensor.header['ROTANGLE'] = sensor.header['ROTANG']
-            except KeyError as e:
-                raise(e)
+
+        sensor.header['RAFTBAY'] = raft
+        sensor.header['CCDSLOT'] = ccdslot
+
+        sensor.header['RASTART'] = sensor.header['RA_DEG']
+        sensor.header['DECSTART'] = sensor.header['DEC_DEG']
+        sensor.header['ROTSTART'] = sensor.header['ROTANG']
+
+        sensor.header['RA'] = sensor.header['RA_DEG']
+        sensor.header['DEC'] = sensor.header['DEC_DEG']
+        sensor.header['ROTPA'] = sensor.header['ROTANG']
+        sensor.header['ROTPOS'] = sensor.header['ROTANG']
```

Comment:
Same question above.

Reply:
As above - removed `ROTPOS` for redundancy (I tested, it wasn't needed for ingestion or ISR)